extensions/2.0/Khronos/KHR_texture_astc/README.md

```diff
@@ -0,0 +1,232 @@
+<!--
+Copyright 2024 The Khronos Group Inc.
+SPDX-License-Identifier: LicenseRef-KhronosSpecCopyright
+-->
+
+# KHR_texture_astc
+
+## Contributors
+
+- Wasim Abbas [@wasimabbas-arm](https://github.com/wasimabbas-arm)
+- Jan-Harald Fredriksen [@janharaldfredriksen-arm](https://github.com/janharaldfredriksen-arm)
+- Peter Harris [@solidpixel](https://github.com/solidpixel)
+- Mark Callow [@MarkCallow](https://github.com/MarkCallow)
+- Alexey Knyazev [@lexaknyazev](https://github.com/lexaknyazev)
+
+Copyright 2024 The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension adds the ability to specify textures using (ASTC compressed images)[https://github.com/ARM-software/astc-encoder/blob/main/Docs/FormatOverview.md] in KTX v2 containers. An implementation of this extension can use such images as an alternative to the PNG or JPEG images available in glTF 2.0 for more efficient asset transmission, reducing GPU memory footprint, higher texture throughput and faster execution.
+
+When this extension is used, it's allowed to use value `image/ktx2` for the `mimeType` property of images that are referenced by the `source` property of `KHR_texture_astc` texture extension object.
+
+At runtime, engines can directly use the ktx images unlike KHR_texture_basisu which requires transcoding to other compressed formats. This also means you get support for all ASTC block formats and quality presets.
```

Comment:
We should not directly refer to Basis here. Suggested replacement:
> When running on supported platforms, engines can directly upload ASTC data without transcoding it to other compressed formats. This means that all ASTC block sizes and quality presets are supported unlike with other texture extensions that focus on portability of compressed texture payloads.

> Platforms that do not have hardware ASTC support would have to either use other texture sources or decompress ASTC data manually.

Reply:
Fixed in https://github.com/KhronosGroup/glTF/pull/2369/commits/b5717f9ffc6285701a87791f68f42d9d47712430